app/model_config.py

```diff
@@ -0,0 +1,173 @@
+import os
+
+# This is directory tree structure for models and their metadata inside of
+# Google Cloud Storage. The top-level item (e.g. "bert_classification_models")
+# is called the 'bucket'.
+# .
+# └── bert_classification_models
+#     └── multilabel
+#         └── UPR_2percent_ps0
+#             └── saved_model
+#                 ├── label.vocab
+#                 └── 1573031002
+#                     ├── saved_model.pb
+#                     └── variables
+#                         ├── variables.data-00000-of-00001
+#                         └── variables.index
+#         └── UPR_2percent_ps512
+#             └── saved_model
+#                 ├── label.vocab
+#                 └── 1572866595
+#                     ├── saved_model.pb
+#                     └── variables
+#                         ├── variables.data-00000-of-00001
+#                         └── variables.index
+
+
+IN = {
```

Comment:
doc - why hardcode all this?

Reply:
I'll at least move it to a JSON file (/static/model_config.json) for now; agree that there's still room for improvement.